subgraphs/hop-protocol/protocols/hop-protocol/src/L2/amm.ts

```diff
@@ -0,0 +1,356 @@
+import { SDK } from "../../../../src/sdk/protocols/bridge";
+import { TokenPricer } from "../../../../src/sdk/protocols/config";
+import {
+  TokenInitializer,
+  TokenParams,
+} from "../../../../src/sdk/protocols/bridge/tokens";
+import {
+  BridgePermissionType,
+  BridgePoolType,
+} from "../../../../src/sdk/protocols/bridge/enums";
+import { BridgeConfig } from "../../../../src/sdk/protocols/bridge/config";
+import { Versions } from "../../../../src/versions";
+import { NetworkConfigs } from "../../../../configurations/configure";
+import {
+  Address,
+  BigDecimal,
+  BigInt,
+  log,
+  dataSource,
+  Bytes,
+} from "@graphprotocol/graph-ts";
+import {
+  TokenSwap,
+  AddLiquidity,
+  RemoveLiquidity,
+  RemoveLiquidityOne,
+} from "../../../../generated/HopL2Amm/L2_Amm";
+import { Token } from "../../../../generated/schema";
+import { getUsdPricePerToken, getUsdPrice } from "../../../../src/prices/index";
+import { bigIntToBigDecimal } from "../../../../src/sdk/util/numbers";
+import {
+  BIGINT_TEN_TO_EIGHTEENTH,
+  SIX,
+} from "../../../../src/sdk/util/constants";
+import { updateAMMTVE } from "../../../../src/sdk/util/tokens";
+
+class Pricer implements TokenPricer {
+  getTokenPrice(token: Token): BigDecimal {
+    const price = getUsdPricePerToken(Address.fromBytes(token.id));
+    return price.usdPrice;
+  }
+
+  getAmountValueUSD(token: Token, amount: BigInt): BigDecimal {
+    const _amount = bigIntToBigDecimal(amount, token.decimals);
+    return getUsdPrice(Address.fromBytes(token.id), _amount);
+  }
+}
+
+const conf = new BridgeConfig(
+  "0x03d7f750777ec48d39d080b020d83eb2cb4e3547",
```

Comment:
Can we make this a constant in a `constants.ts` file?

Reply:
Okay, I'm importing a config now, which fixes this.